lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -468,7 +468,7 @@ def store_at_0
           end
 
           def update_positions
-            old_position = send("#{position_column}_was").to_i
+            old_position = send("#{position_column}_was") || bottom_position_in_list + 1
```

Comment:
Thanks @StoneFrog, that's a great explanation :) Thanks for putting my mind at rest. I'll merge this.